client/src/Components/Pages/Dashboard/MyActivety/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './style.css';
+import loading from './images/loading.png';
+import request from './images/request.png';
+
+export default function index (props){
+  const{activites}=props;
+  return(
+  <React.Fragment>
```

Comment:
This React.Fragment has only a single child, (div.my-activity), so you don't really need it.  React.Fragment is most useful when you have multiple child elements that need to be wrapped with something.

Reply:
Yes, I started with the React.Fragment, but when I started to style the component I added the <div>.
I will modify it.